src/app/(admin)/new_dashboard/guides/hooks/useSearchGuides.ts

```diff
@@ -1,9 +1,11 @@
 import { useState } from "react";
 import { Guide } from "@/api/types/Guide";
+import { SelectChangeEvent } from "@mui/material";
 
 export const useSearchGuides = (initialGuides: Guide[]) => {
   const [searchQuery, setSearchQuery] = useState("");
   const [selectedTag, setSelectedTag] = useState("All");
+  const [sortCriteria, setSortCriteria] = useState<"rating" | "date">("rating");
```

Comment:
make this more abstract, what happens if tomorrow we want to add another cateogry?

Reply:
imported type SortCriteria from `types`. default sorting option is by Date